ground/src/main/java/com/google/android/ground/ui/datacollection/DataCollectionFragment.kt

```diff
@@ -49,11 +46,19 @@ class DataCollectionFragment : AbstractFragment() {
         super.onCreateView(inflater, container, savedInstanceState)
         val binding = DataCollectionFragBinding.inflate(inflater, container, false)
 
+        val args = getDataCollectionFragmentArgs()
```

Comment:
Would it be possible to add a test for this behavior? (load submission on fragment creation)

Reply:
Done!